src/tools/scripts/run_and_test_website.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+######################################
+## Custom Web Almanac script        ##
+######################################
+#
+# This script installs all the required dependencies needed to run the
+# Web Almanac website providing you have python and node installed.
+#
+# It also runs our tests to ensure the website is working for all pages.
+#
+# It is used by various GitHub actions to build and test the site.
+#
+
+# exit when any command fails instead of trying to continue on
+set -e
```

Comment:
You might want to also set `x` flag so each interpolated command is printed when it is run. This same thing can be done in the other Shell script too to see why certain commands might be failing.

```suggestion
set -e
set -x
```

Reply:
Done!